iis-wf/iis-wf-preprocessing/src/main/resources/eu/dnetlib/iis/wf/preprocessing/main/oozie_app/workflow.xml

```diff
@@ -531,6 +531,14 @@
 					<name>output_document_to_research_initiatives</name>
 					<value>${workingDir}/transformers_concept_to_researchinitiatives/out</value>
 				</property>
+				<property>
+                    <name>output_report_relative_path</name>
+                    <value>documentWithAtLeastOneConcept</value>
+                </property>
+                <property>
+                    <name>output_report_record_written_counter_name</name>
+                    <value>export.referenceExtraction.documentWithAtLeastOneConcept</value>
```

Comment:
The convention introduced by Krzysiek is `docsWithAtLeastOne*` which I have followed in one of the modules. I guess it will be good to keep all those properties in sync.